Lib/aifc.py

```diff
@@ -451,23 +451,31 @@ def readframes(self, nframes):
     #
 
     def _alaw2lin(self, data):
-        import audioop
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', category=DeprecationWarning)
+            import audioop
```

Comment:
How about using `import_deprecated` on all these like the other test files below?

```suggestion
        audioop = import_deprecated("audioop")
```

Reply:
This is not a test file. It's the ``aifc`` module. We cannot use any test helpers in stdlib modules. The ``test`` package is optional. Most distros have it split into a separate distro package to conserve disk space.